dev/archery/archery/crossbow/core.py

```diff
@@ -696,7 +696,7 @@ def get_version(root, **kwargs):
 
     # query the calculated version based on the git tags
     kwargs['describe_command'] = (
-        'git describe --dirty --tags --long --match "apache-arrow-[0-9].*"'
+        'git describe --dirty --tags --long --match "apache-arrow-[0-9]*.*"'
```

Comment:
I would mandate at least one digit
```suggestion
        'git describe --dirty --tags --long --match "apache-arrow-[0-9]+.*"'
```

Reply:
I don't think that works as is git uses glob not regex. I did try that first :) :
```
$ git describe --dirty --tags --long --match "apache-arrow-[0-9]+.*"
fatal: No names found, cannot describe anything.
```

Reply:
Ahah, I see. Git being git, as usual...